### PR TITLE
[daikin] Support AUTO fan mode with Airbase

### PR DESCRIPTION
**Pull request by jimtng**
Airbase supports AUTO fan mode as reported in https://community.openhab.org/t/daikin-binding-fan-error/96129

This is done by setting f_rate=0,f_airside=0,f_auto=0

It appears that this is also allowed in other Airbase compatible systems that don't otherwise allow such setting via the wall controller.

### CODE REVIEW

File: bundles/org.openhab.binding.daikin/src/main/java/org/openhab/binding/daikin/internal/api/airbase/AirbaseEnums.java [hunk 1]
```diff
@@ -60,6 +60,7 @@ public static AirbaseMode fromValue(int value) {
 
     public enum AirbaseFanSpeed {
         // level,f_auto,f_airside
+        AUTO         (0, false, false),
```
Comment (cpmeister) [inline bundles/org.openhab.binding.daikin/src/main/java/org/openhab/binding/daikin/internal/api/airbase/AirbaseEnums.java:63]: This class has a field called `auto`. Why isn't it set to true for this new enum value?

Comment (jimtng) [reply bundles/org.openhab.binding.daikin/src/main/java/org/openhab/binding/daikin/internal/api/airbase/AirbaseEnums.java:63]: Could you rephrase the question please?

Comment (jimtng) [reply bundles/org.openhab.binding.daikin/src/main/java/org/openhab/binding/daikin/internal/api/airbase/AirbaseEnums.java:63]: Are you referring to f_auto? f_auto applies to AUTO_LEVEL_N. When the Fan is set to "auto" (not auto_level_n), f_auto is set to false (0). That's just how the controller works.

Comment (cpmeister) [reply bundles/org.openhab.binding.daikin/src/main/java/org/openhab/binding/daikin/internal/api/airbase/AirbaseEnums.java:63]: understood

**Comment by cpmeister** [comment]
@caffineehacker Does this look good to you?

**Comment by jimtng** [comment]
> @caffineehacker Does this look good to you?

he hasn't been around for a while. @psmedley has asked to be added as the maintainer

**Comment by psmedley** [comment]
> > @caffineehacker Does this look good to you?
> 
> he hasn't been around for a while. @psmedley has asked to be added as the maintainer

LGTM

**Comment by cpmeister** [comment]
Well, fine then. 
@psmedley Does this PR look good to you? Also, can you submit a PR with an update to the CODEOWNERS file that adds you as another maintainer for this project? That is if you really want to take on the role.

**Comment by cpmeister** [comment]
Jeez, you responded faster than I could summon you.
